launch/house_simulation.launch

```diff
@@ -20,6 +20,10 @@
     <arg name="map_file" value="$(arg map_file)"/>
   </include>
 
+  <include file="$(find fetch_moveit_config)/launch/move_group.launch">
```

Comment:
This is the correct thing to add, but not quite the right place. Since MoveIt is something we want across all simulations (not just house), it needs to be in the `common` sub-launchfile.